app/templates/select-item-layout.hbs

```diff
@@ -1,5 +1,11 @@
 {{#if view.content.isGroupOption}}
   {{view.content.name}}
 {{else}}
-  {{yield}}
```

Comment:
I have some concern here. This changes behavior of this template. People who use this with some custom content is now expect the change.